app/helpers/solidus_viabill/api/checkout_helper.rb

```diff
@@ -33,13 +33,13 @@ def build_checkout_request_body(order, payment_method_id, frontend)
           }
         }
         request_body[:sha256check] = gateway.generate_signature(
-          gateway.api_key,
```

Comment:
Is possible to add a test to prevent that this would happen again in the future?